cloud/microsoft/office365/skype/mode/devicesusage.pm

```diff
@@ -58,7 +62,7 @@ sub custom_active_threshold {
 sub custom_active_output {
     my ($self, %options) = @_;
 
-    my $msg = sprintf("Active users on %s : %d/%d (%.2f%%)",
+    my $msg = sprintf("Active devices on %s : %d/%d (%.2f%%)",
```

Comment:
Can we keep the active users? And add an "active devices" instead.

Reply:
We already have the active / inactive users metrics in `useractivity` mode.
I then thought useless to duplicate the metric, which would then be visible 2 times at different places in Centreon, Stream Connector Scripts' targets such as InfluxDB etc... There's no need to have 2 different metrics pointing to the same thing...
I also found more interesting to report devices, in a `deviceusage` mode, properly separating things, making again the whole thing more readable.